server/views/pages/approve/view.njk

```diff
@@ -13,7 +13,15 @@
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-three-quarters">
-            <h1 class="govuk-heading-l">Approve a licence for {{ licence.forename }} {{ licence.surname }}</h1>
+            {% if licence.typeCode === 'AP' %}
+                <h1 class="govuk-heading-l">Approve licence for {{ licence.forename }} {{ licence.surname }}</h1>
+            {% elseif licence.typeCode === 'AP_PSS' %}
+                <h1 class="govuk-heading-l">Approve licence and post sentence supervision order for {{ licence.forename }} {{ licence.surname }}</h1>
+            {% elseif licence.typeCode === 'PSS' %}
+                <h1 class="govuk-heading-l">Approve post sentence supervision order for {{ licence.forename }} {{ licence.surname }}</h1>
+            {% else %}
+                <h1 class="govuk-heading-l">Approve licence type {{ licence.typeCode }} for {{ licence.forename }} {{ licence.surname }}</h1>
```

Comment:
Is this possible?

Reply:
Not at present. Fingers crossed - unless its gone wrong.